l10n_it_sdi_channel/readme/CONFIGURE.rst

```diff
@@ -9,13 +9,6 @@ Creare un nuovo canale di tipo PEC (unico supportato per ora) e indicare:
 
 Dopo il primo invio, lo SdI risponderà segnalando l'indirizzo da utilizzare per gli invii successivi, da inserire nella configurazione del server PEC dedicato.
 
-Nella configurazione del server, sezione "PEC e fattura elettronica", selezionare la casella "Server PEC e-fattura".
```

Comment:
Come mai qui sopra ci sono le istruzioni per creare un canale SdI di tipo PEC? Dopo le modifiche in questa PR, installando solo questo modulo non sarà possibile farlo.
Credo che anche queste sarebbero da spostare nel modulo dedicato al canale PEC, che ne pensi?

Reply:
Ho lasciato per avere un esempio di creazione canale, magari esplicitiamo meglio

Reply:
Ah ok, allora forse si può lasciare qualcosa, tipo dov'è il menu dei canali SdI; però l'esempio con proprio i parametri specifici del canale PEC secondo me andrebbe nel modulo dedicato alla PEC (così anche lì hai un esempio).
Se lo spostiamo, qui si può lasciare semplicemente l'indicazione di guardare nel modulo **l10n_it_fatturapa_pec** per un esempio dettagliato.

Reply:
Modifico. Hai un suggerimento su come mantenere i 2 commit, uno per modulo?

Reply:
Altra cosa, dovrei anche incrementare la versione sul `__manifest__.py` di `l10n_it_sdi_channel`?

Reply:
Spero di non aver fatto casino 😅

Reply:
> Modifico. Hai un suggerimento su come mantenere i 2 commit, uno per modulo?

1. Modifichi il `CONFIGURE` di sdi_channel e metti le modifiche in un commit di fixup eseguendo
    `git commit --fixup=<hash del commit di sdi_channel>`
2. Modifichi il `CONFIGURE` di fatturapa_pec e metti le modifiche in un commit di fixup eseguendo
    `git commit --fixup=<hash del commit di fatturapa_pec>`
3. Esegui il rebase di tutti i tuoi commit, in modo che applichi automaticamente i fixup con
    `git rebase --interactive --autosquash 14.0`
    Se così ti modifica troppi commit, probabilmente è perché il tuo `14.0` non è aggiornato o il branch su cui stai lavorando ha bisogno di un `rebase`; nel caso puoi aggiornare `14.0` e fare `rebase` del branch su cui stai lavorando,
    oppure sostituire `14.0` con `<hash del commit più vecchio>^` (nota il `^` per prendere il commit precedente a quello indicato)
4. Verifichi che sia tutto a posto in locale, poi modifichi la PR eseguendo
    `git push -f`

Esistono anche altri modi, ma questo secondo me è il più semplice.

> Altra cosa, dovrei anche incrementare la versione sul `__manifest__.py` di `l10n_it_sdi_channel`?

Dovrebbe essere incrementato automaticamente, ma se vuoi puoi aggiornarlo nella PR

> Spero di non aver fatto casino :sweat_smile:

Vedo che hai rimosso le istruzioni da sdi_channel, ma non le vedo ri-aggiunte in fatturapa_pec quindi temo tu ti sia perso qualche pezzo :)

Reply:
> > Dovrebbe essere incrementato automaticamente, ma se vuoi puoi aggiornarlo nella PR
> 
> Si, ma avendo uno script di migrazione su `l10n_it_fatturapa_pec` ho incrementato la versione sul manifest, di conseguenza lo devo fare anche su `l10n_it_sdi_channel`, corretto?

Secondo me non è necessario, però se preferisci puoi farlo.


> > Vedo che hai rimosso le istruzioni da sdi_channel, ma non le vedo ri-aggiunte in fatturapa_pec quindi temo tu ti sia perso qualche pezzo :)
> 
> 🤦🏻‍♂️. Ora dovrebbero esserci ..

Sì ora mi pare essere a posto, grazie!